src/NzbDrone.Core/MediaFiles/TrackImport/ImportApprovedEpisodes.cs

```diff
@@ -12,10 +12,10 @@
 using NzbDrone.Core.Qualities;
 using NzbDrone.Core.Download;
 using NzbDrone.Core.Extras;
-using NzbDrone.Core.MediaFiles.TrackImport;
 
-namespace NzbDrone.Core.MediaFiles.EpisodeImport
+namespace NzbDrone.Core.MediaFiles.TrackImport
 {
+    [Obsolete("Used by Sonarr, not by Lidarr")]
```

Comment:
I guess this means things break if we try to take it out now?

Reply:
Yep, will be able to pull out when completed handling goes in after this pr